config/routes.rb

```diff
@@ -16,6 +16,7 @@
   get '/travel_game', to: 'journey#game'
   get '/restart_game', to: 'journey#restart', as: 'restart_game'
   get '/help', to: 'journey#help'
+  get 'admin/activate_avatar', to: 'admin/avatars#activate'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.